tardis/opacities/opacities.py

```diff
@@ -278,7 +278,8 @@ def compton_opacity_partial(energy, fraction):
 
 @njit(**njit_dict_no_parallel)
 def compton_opacity_calculation(energy, electron_density):
```

Comment:
This should have been moved?

Reply:
nope - I decided against it for this PR. But in general yes.